assets/src/edit-story/components/panels/design/filter/filter.js

```diff
@@ -32,11 +32,11 @@ import {
   OverlayPreset,
   OverlayType,
 } from '../../../../utils/backgroundOverlay';
```

Comment:
Renaming the underlying functionality in here can be done when generalizing filters to also work for non-background media in #6526.